ktor-client/ktor-client-tests/jvm/src/io/ktor/client/tests/utils/tests/CloseableScope.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.tests.utils.tests
+
+import java.io.*
+
+class CloseableScope : Closeable {
```

Comment:
Minor: `Scope` in the name makes me think about `CoroutineScope`. Can you think of some other name?

Reply:
Do you have any advice?

Reply:
`ClosableGroup` or something similar? But it's fine if you leave it like that